### PR TITLE
Improve coverage

### LINKED ISSUE #20: Improve test coverage
Currently the coverage is at 63%, we should aim to >90% or why not 100%.

### PR DESCRIPTION
**Pull request by matextrem**
Closes #20 

This PR improve coverage of API methods + Some utils functions

Coverage reaches `≈85%`

The lines were not covered are related to some edge cases really difficult to test. We are not using those in our API methods.

### CODE REVIEW

File: src/api/cow/cow.spec.ts [hunk 1]
```diff
@@ -215,6 +291,30 @@ test('Invalid: Get last 5 trades for an unexisting trader ', async () => {
...
+      kind: 'buy',
+      buyAmountAfterFee: '1234567890',
+      sellToken: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+      buyToken: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+      from: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+      receiver: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+      appData: '0x0000000000000000000000000000000000000000000000000000000000000000',
+      validTo: 0,
+      partiallyFillable: false,
+    }),
```
Comment (alfetopito) [inline src/api/cow/cow.spec.ts:310]: Shouldn't this bit of the body be equivalent to the string-fied QUOTE_RESPONSE constant?

At a bare minimum, could you use that to fill in the values here rather than repeating them

**Review by alfetopito — APPROVED** [review]
Thanks for this!

The only comment I have is to re-use constants if possible to avoid copy paste of values in the tests which makes it harder in case we need to update something later on

**Review by ramirotw — APPROVED** [review]
